### DISCUSSION TITLE
Issue ideas - Milestone 3

### DISCUSSION THREAD

**Discussion by krigubir** [ksvedal/uu-chrome-extension#35] [Ideas]
Issue ideas relevant for milestone 3

**Comment by andreascnits** [comment]
Skip to main content

As long as there is content on the page that shows globally,  there should be a button available to skip to main content

https://www.uutilsynet.no/veiledning/tastaturnavigasjon/37

**Comment by krigubir** [comment]
1.3.5 Identifiser formål med inndata 

Gjøre det enklere å fylle inn skjemadata ved hjelp av autocomplete-attribute i HTML. I tillegg skal datatypen være spesifisert i koden (e.g. email, navn, mobilnummer i de ulike skjemafletene)

https://www.uutilsynet.no/wcag-standarden/135-identifiser-formal-med-inndata-niva-aa/142
